java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigquery.java

```diff
@@ -128,9 +136,17 @@ public void runTemplate() {
           break;
         default:
           throw new IllegalArgumentException(
-              "Currenlty avro, parquet and csv are the only supported formats");
+              "Currently avro, parquet and csv are the only supported formats");
       }
 
+      if (bqTempTable == null || bqTempQuery == null) {
```

Comment:
So rather than setting default values when bqTempTable or bqTempQuery are not provided, I think it would be cleaner to only apply spark SQL when these are set

Reply:
Done